Source/MediaInfo/Tag/File_Exif.cpp

```diff
@@ -1003,6 +1321,9 @@ void File_Exif::Streams_Finish()
     const auto Infos_Exif_It = Infos.find(Kind_Exif);
     const auto Infos_GPS_It = Infos.find(Kind_GPS);
     const auto Infos_Interop_It = Infos.find(Kind_Interop);
+    const auto Infos_MakernoteApple_It = Infos.find(Kind_MakernoteApple);
+    const auto Infos_MakernoteNikon_It = Infos.find(Kind_MakernoteNikon);
+    const auto Infos_MakernoteSony_It = Infos.find(Kind_MakernoteSony);
```

Comment:
I want to change that for others in the future, for the new items please move these lines just before the "if" for less diff.